### PR TITLE
Add integrating with vizia and option views docs

### PR DESCRIPTION
**Pull request by garann**
Please have a look at the option views doc, as this could benefit from the expertise of the people who built or have worked on these individual views.

### CODE REVIEW

**Comment by jonathanmiles** [comment]
The options views aren't particularly fresh in my mind but this looks good (from what I remember, and bearing in mind that I'm not familiar with everything).

I did pause for thought around `MinimalControlsOptionsView`. It takes an `inputTypes` option and so I _think_ tends to get used by being extended, rather than as something standalone - so it just being the basis for other things might be worth emphasising.

I think that in turn could lead to a clearer description for `QueryControlsOptionsView`, because it currently sounds a bit like both classes have query controls... whereas it's more like the latter is extending the former and configuring it to only have controls for selecting individual queries (rather than configuring it to also allow query group selection).

**Comment by garann** [comment]
Thanks, @jonathanmiles, that's super helpful. I spent a long time trying to figure out the relationship between those, but missed that `inputTypes` is required. :) I'll update to take advantage of this new info.

File: docs/npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,49 @@
+0 info it worked if it ends with ok
```
Comment (jonathanmiles) [inline docs/npm-debug.log]: Not sure if you already have this in hand but this file might be one to ignore.

Comment (garann) [inline docs/npm-debug.log]: Accurate, and I did not. Thanks!

**Pull request merged**